Testscripts/Windows/PERF-STORAGE-MULTIDISK-RAID0-FIO.ps1

```diff
@@ -234,11 +234,19 @@ function Main {
             -RunInBackground -runAsSudo
 
         Write-LogInfo "Monitoring test run..."
+        $ForceExitCount = 0
```

Comment:
This value needs to reset in While loop, if the $current status changes from "Doing forceful exit of this job" --> "next iteration" . And change the variable name to - `$FioStuckCounter`

Reply:
done